conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```
@@ -11,7 +11,20 @@ jobs:
     fetchDepth: {{ clone_depth }}
 {%- endif %}
   - script: |
-         rm -rf /opt/ghc
+         sudo mkdir -p /opt/empty_dir
```

```suggestion
         sudo mkdir -p /opt/empty_dir || true
```

Do we want to add this as well? I don't think `mkdir -p` can fail unless there is a permission issue (which there shouldn't be with `sudo` here)

---

It wouldn't hurt, I suppose. We can also add the `continueOnError` [property](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-script?view=azure-pipelines) to the script (after `displayName`) so that failure does not stop the build. I don't want to be the person that breaks all the builds. 😄

---

Exactly, I am very excited to get this merged in but oh boy will it be disruptive if there is a mistake :see_no_evil: